openstack/tools/upload_octavia_amphora_image.sh

```diff
@@ -1,2 +1,23 @@
-#!/bin/bash -eu
-juju run octavia-diskimage-retrofit/0 retrofit-image
+#!/bin/bash
+
+set -e -u
+
+basedir=$(realpath $(dirname $0))
+source ${basedir}/../novarc
+
+set -x
+
+ID=$(openstack image show --format value --column id jammy)
```

Comment:
My only enhancement here would be to have the script accept an optional argument - focal/jammy/noble/etc, but default to jammy if none is given.

Reply:
I added a command line option for the series.